qa/smoke-test-http/src/test/java/org/opensearch/http/HttpCatIT.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.http;
+
+import org.apache.hc.core5.http.ParseException;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.junit.BeforeClass;
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+import org.opensearch.client.RestClient;
+import org.opensearch.test.OpenSearchIntegTestCase;
+
+import java.io.IOException;
+
+import static org.apache.hc.core5.http.HttpStatus.SC_OK;
+import static org.hamcrest.Matchers.containsString;
+
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.SUITE, supportsDedicatedMasters = false, numDataNodes = 5, numClientNodes = 0)
+public class HttpCatIT extends HttpSmokeTestCase {
+
+    @BeforeClass
+    public static void doNotSetAvailableProcessors() {
+        System.setProperty("opensearch.set.netty.runtime.available.processors", "false");
+    }
```

Comment:
Does this get cleaned up or will it end up having side effects for other tests?

Reply:
It seems having not side effects for other tests, it just decide how many `node.processors` each node has.

We create 5 nodes in the jvm, each node has the different `node.processors`:
https://github.com/opensearch-project/OpenSearch/blob/087355f0ee676064ea409ed68090b33e568ea941/test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java#L539
and all of the five nodes will call `Netty4Utils.setAvailableProcessors()`,  if we don't set it, it must throw `IllegalStateException`.

Reply:
Maybe `doNotSetAvailableProcessors` do affect other tests, It's also set in `build.gradle`, I try to delete the system setting.